### PR TITLE
Add unit tests for mappers #3

### PR DESCRIPTION
**Pull request by bhegmanns**
Hi, I made a little small test for BottomShield

### CODE REVIEW

File: shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package nf.fr.k49.shielddb.core;
+
+public class ShieldDbBuilderForStringTest implements BaseShieldDBBuilderTest<String>{
+}
```
Comment (kuroidoruido) [inline shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java]: Maybe can you add another test class with a custom object, in real use case we will mainly store custom object (maybe a simple User(firstname,lastname,birthdate))

File: shielddb-demo-app/src/main/java/nf/fr/k49/shielddb/demo/app/App.java [hunk 2]
```diff
@@ -134,10 +133,15 @@ private static void model1UserAtomic() throws IOException, InterruptedException
...
-
+		executor.shutdownNow();
 		System.out.println(userDb.size() + " -> " + userDb.toString());
 	}
 
+
+
+
+
+
```
Comment (kuroidoruido) [inline shielddb-demo-app/src/main/java/nf/fr/k49/shielddb/demo/app/App.java]: can you remove these blank lines?

**Review by kuroidoruido — CHANGES_REQUESTED** [review]

File: shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java (hunk 1 above)
Comment (bhegmanns) [reply shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java]: No problem. What about the Testclasses for String and Integer? Delete?

Comment (kuroidoruido) [reply shielddb-core/src/test/java/nf/fr/k49/shielddb/core/ShieldDbBuilderForStringTest.java]: I think you can keep it, they can represent some use cases too :slightly_smiling_face:

**Comment by kuroidoruido** [comment]
@bhegmanns  I think you can rebase on master to be up to date as I merged your other PR :slightly_smiling_face:

**Comment by bhegmanns** [comment]
Sorry for the latency; I was too busy during the week. But coming weekend, I'll get time for adding a proper Pojo-example and merging the master ;)

**Comment by bhegmanns** [comment]
master is merged :)
and add new test

**Pull request merged**